src/Rector/Class_/CommandDescriptionToPropertyRector.php

```diff
@@ -108,33 +111,42 @@ private function resolveCommandDescription(Class_ $class): ?Node
     private function resolveCommandDescriptionFromSetDescription(Class_ $class): ?Node
     {
         $commandDescription = null;
-
+        $classMethod = $class->getMethod('configure');
+        if (! $classMethod instanceof ClassMethod) {
+            return null;
+        }
         $this->traverseNodesWithCallable($class->stmts, function (Node $node) use (&$commandDescription) {
```

Comment:
Oh, it need to be `$classMethod->stmts`:

```suggestion
        $this->traverseNodesWithCallable((array) $classMethod->stmts, function (Node $node) use (&$commandDescription) {
```

Reply:
++